org-owners-guide.md

```diff
@@ -0,0 +1,52 @@
+# Kubernetes Github Organization Guide
+
+The Kubernetes project leverages multiple GitHub organizations to store and
+organize code. This guide contains the details on how to run those organizations
+for CNCF compliance and for the guidelines of the community.
+
+## Organization Naming
+
+Kubernetes managed organizations should be in the form of `kubernetes-[thing]`.
+For example, [kubernetes-client](https://github.com/kubernetes-client) where the
+API clients are housed.
+
+Prior to creating an organization please contact the steering committee for
+direction and approval.
+
+Note: The CNCF, as part of the Linux Foundation, holds the trademark on the
+Kubernetes name. All GitHub organizations with Kubernetes in the name should be
+managed by the Kubernetes project or use a different name.
+
+## Transferring Outside Code Into A Kubernetes Organization
+
+Due to licensing and CLA issues, prior to transferring software into a Kubernetes
+managed organization there is some due diligence that needs to occur. Please
+contact the steering committee and CNCF prior to moving any code in.
+
+It is easier to start new code in a Kubernetes organization than it is to
+transfer in existing code.
+
+## Current Organizations In Use
+
+The following organizations are currently known to be part of the Kubernetes
+project:
```

Comment:
The steering committee will be pushing out a doc soon regarding organizational structure which will likely cause the structure below to be reformed.

Reply:
Does that need to block this PR? Can we update the doc after those changes come out?

Reply:
I'd put a clause and change the content saying "this is all subject to change", but here is the current state, then lgtm from me.

Reply:
Added that clause